src/renderers/sigma.renderers.canvas.js

```diff
@@ -402,11 +396,9 @@
    * @return {sigma.renderers.canvas} Returns the instance itself.
    */
   sigma.renderers.canvas.prototype.clear = function() {
```

Comment:
I did [this jsperf test](http://jsperf.com/canvas-clearrect-vs-width/5) on many browsers. 

More info on the PR: https://github.com/Linkurious/linkurious.js/pull/223